model/metrics/system-metrics.yaml

```
@@ -77,6 +77,14 @@ groups:
     unit: "By"
     attributes: []
 
+  - id: metric.system.memory.shared
+    type: metric
+    metric_name: system.memory.shared
+    stability: experimental
+    brief: "Reports shared memory."
```

Can it be a bit more specific?
```suggestion
    brief: "Reports shared memory usage."
```